src/libraries/Native/Unix/System.Globalization.Native/pal_calendarData.c

```diff
@@ -5,12 +5,22 @@
 #include <assert.h>
 #include <stdlib.h>
 #include <string.h>
-#include <strings.h>
-
 #include "pal_locale_internal.h"
 #include "pal_errors_internal.h"
 #include "pal_calendarData.h"
 
+#if defined(TARGET_UNIX)
+#include <strings.h>
+
+#define STRING_COPY(destination, numberOfElements, source) \
+    strncpy(destination, source, ULOC_FULLNAME_CAPACITY); \
```

Comment:
kind of a nit: It seems a bit odd to hard-code the size here to `ULOC_FULLNAME_CAPACITY` yet in the macro below is uses `numberOfElements` as the value for the max size of `destination`.